src/modules/location/selectors.ts

```diff
@@ -50,6 +50,18 @@ export const getCollectionId = (state: RootState) => {
   return result ? result.params.collectionId : null
 }
 
+const thirdPartyCollectionIdMatchSelector = createMatchSelector<
+  RootState,
+  {
+    collectionId: string
+  }
+>(locations.thirdPartyCollectionDetail())
+
+export const getThirdPartyCollectionId = (state: RootState) => {
+  const result = thirdPartyCollectionIdMatchSelector(state)
+  return result ? result.params.collectionId : null
+}
+
```

Comment:
Can you add a test for this new selector?

Reply:
Yes!